examples/resources/awscc_apigatewayv2_route/apigatewayv2_route.tf

```diff
@@ -0,0 +1,18 @@
+resource "awscc_apigatewayv2_api" "example_api" {
+  name                       = "example-websocket-api"
+  protocol_type              = "WEBSOCKET"
+  route_selection_expression = "$request.body.action"
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
+
+resource "awscc_apigatewayv2_route" "example_route" {
```

Comment:
this example is incomplete with the target route, @catlike  - I suggest to add Mock target and refer it

i.e.
```
resource "aws_apigatewayv2_integration" "example" {
  api_id           = awscc_apigatewayv2_api.example_api.id
  integration_type = "MOCK"
}
```